ouroboros-consensus/test/consensus-test/Test/Consensus/HardFork/Infra.hs

```diff
@@ -111,6 +113,7 @@ genEraParams = do
     eraEpochSize  <- EpochSize         <$> choose (1, 10)
     eraSlotLength <- slotLengthFromSec <$> choose (1, 5)
     eraSafeZone   <- genSafeZone
+    eraGenesisWin <- GenesisWindow     <$> choose (1, 10)
```

Comment:
Does it make sense to choose a genesis window without considering values chosen for other parameters?